### PR TITLE
Cockpit 234

### PR DESCRIPTION
**Pull request by mvollmer**
https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-234.html

### CODE REVIEW

File: _posts/2020-12-09-cockpit-234.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Cloning creates a new VM that uses its own disk image for storage, but most of the clone’s configuration and stored data is identical to the source VM. This makes it possible to prepare a number of VMs optimized for a certain task without the need to optimize each VM individually.
+
+![screenshot of the machine cloning](/images/machines-cloning.png)
+
+## The dashboard has been removed
+
+The very limited functionality of the dashboard does not justify its
+continued maintenance, in our opinion.  The Shell can connect to
+remote hosts and Grafana etc are much better at monitoring multiple
+machines.
```
Comment (marusak) [inline _posts/2020-12-09-cockpit-234.md:47]: Due to it's limited functionality the dashboard page has been removed. Additional hosts can be managed through shell without any need for a separate package. The graphs part was not providing enough information to be kept as for detailed metrics a separate page is available on the overview page and for multihost monitoring other tools (e.g. Grafana) provide more useful data.

Comment (mvollmer) [reply _posts/2020-12-09-cockpit-234.md:47]: :+1:

**Comment by garrett** [comment]
Thanks for your first draft! I pushed up a whole bunch of changes. Hopefully it's more clear now.

I'm working on the images now and will push that up soon too. :wink:

**Comment by garrett** [comment]
GitHub hasn't changed your branch. Here's a preview from mine:
Screenshot_2020-12-10 Cockpit 234 and Cockpit-Podman 26 — Cockpit Project

**Review by marusak — APPROVED** [review]
Nice, thanks!

**Comment by garrett** [comment]
I hope I'm technically accurate in the rewrites. Please let me know if I got anything wrong. :wink:

**Comment by KKoukiou** [comment]
@garrett 'with the exception of the few configuration changes that need to differ.' that was not really correct but I just saw that.
The only thing that changes in the XML is the VM name and UUID, these are not really configuration changes.

**Pull request merged**